index.html

```diff
@@ -0,0 +1,230 @@
+                </div>
+            </div>
+            <br>
+        <div class="box cards1">
+                <div class="blank">
+                    <div class="cardscontainers">
+                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
+                        <ul class="ul1">
+                            <li class="ruby2">Ruby on rails</li>
+                            <li class="css0">css</li>
+                            <li class="java0">JavScript</li>
+                            <li class="html0">html</li>
+                        </ul>
+                        <button type="submit" class="button1">See Project</button>
+                    </div>
+                </div>
+            </div>
+            <br>
+            <div class="box cards1">
+                <div class="blank">
+                    <div class="cardscontainers">
+                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
+                        <ul class="ul1">
+                            <li class="ruby2">Ruby on rails</li>
+                            <li class="css0">css</li>
+                            <li class="java0">JavScript</li>
+                            <li class="html0">html</li>
+                        </ul>
+                        <button type="submit" class="button1">See Project</button>
+                    </div>
+                </div>
+            </div>
+            <br>
+            <div class="box cards1">
+                <div class="blank">
+                    <div class="cardscontainers">
+                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
+                        <ul class="ul1">
+                            <li class="ruby2">Ruby on rails</li>
+                            <li class="css0">css</li>
+                            <li class="java0">JavScript</li>
+                            <li class="html0">html</li>
+                        </ul>
+                        <button type="submit" class="button1">See Project</button>
+                    </div>
+                </div>
+            </div>
+            <br>
+            <div class="box cards1">
+                <div class="blank">
+                    <div class="cardscontainers">
+                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
+                        <ul class="ul1">
+                            <li class="ruby2">Ruby on rails</li>
+                            <li class="css0">css</li>
+                            <li class="java0">JavScript</li>
+                            <li class="html0">html</li>
+                        </ul>
+                        <button type="submit" class="button1">See Project</button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="middle2">
+        <div class="background2">
+                <h2 class="subtitle3">About me</h2>
```

Comment:
- [ ] Kindly fix the about me section to look the same as the Figma design
- Your Design
![Capturedesto](https://user-images.githubusercontent.com/81444338/171762027-5ce84f24-97b9-4762-9f08-5063d6e07565.PNG)

- Figma Design
![Capturefide](https://user-images.githubusercontent.com/81444338/171762149-b5f51ccc-f53c-417d-a467-f411466ce2d5.PNG)